iron-list.html

```diff
@@ -1739,27 +1739,24 @@
     },
 
     _createFocusBackfillItem: function() {
-      var pidx, fidx = this._focusedIndex;
-      if (this._offscreenFocusedItem || fidx < 0) {
+      var fidx = this._focusedIndex;
+      var pidx = this._getPhysicalIndex(fidx);
+
+      if (this._offscreenFocusedItem || pidx == null || fidx < 0) {
```

Comment:
ok, i think i understand now.  the diff is just bigger than i expected (but i now understand why).